python/ray/tune/_structure_refactor.py

```diff
@@ -26,4 +26,4 @@ def warn_structure_refactor(old_module: str, new_module: str, direct: bool = Tru
 
         with warnings.catch_warnings():
             warnings.simplefilter("always")
-            warnings.warn(warning, DeprecationWarning)
+            warnings.warn(warning, DeprecationWarning, stacklevel=3)
```

Comment:
@xwjiang2010 any thoughts on this? 

Previously it would show:
``/Users/rliaw/ray/python/ray/tune/_structure_refactor.py:29: DeprecationWarning: The module `ray.tune.trial_runner` has been moved to `ray.tune.execution.trial_runner` and the old location will be deprecated soon. Please adjust your imports to point to the new location.Example: Do a global search and replace `ray.tune.trial_runner` with `ray.tune.execution.trial_runner`.
```

But now it will show:
```
/Users/rliaw/miniconda3/envs/doc/lib/python3.7/site-packages/tune_sklearn/tune_search.py:15: DeprecationWarning: The module `ray.tune.suggest.bohb` has been moved to `ray.tune.search.bohb` and the old location will be deprecated soon. Please adjust your imports to point to the new location. Example: Do a global search and replace `ray.tune.suggest.bohb` with `ray.tune.search.bohb`.
```

Reply:
looks good to me! learned a new trick today :)